packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressSearchSuggestions.vue

```diff
@@ -163,18 +168,21 @@ export default {
          *
          * @param e
          * @param item
```

Comment:
This param has been removed by the looks of it

Reply:
And the JSDoc will need updating to remove the reference to `item`

Reply:
Fixed.